x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/precision_slider.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiPanel,
+  EuiRange,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { RelevanceTuningLogic } from '../../relevance_tuning_logic';
+
+import { getStepDescription } from './utils';
+
+export const PrecisionSlider: React.FC = () => {
+  const {
+    searchSettings: { precision },
+  } = useValues(RelevanceTuningLogic);
+
+  const { updatePrecision } = useActions(RelevanceTuningLogic);
+
+  const stepDescription = getStepDescription(precision);
+
+  return (
+    <>
+      <EuiTitle size="m">
+        <h2>
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.title',
+            {
+              defaultMessage: 'Precision tuning',
+            }
+          )}
+        </h2>
+      </EuiTitle>
+      <EuiSpacer />
+      <EuiPanel color="subdued" hasShadow={false}>
```

Comment:
@daveyholler Didn't we discuss at some point not using a panel for this?

Reply:
This is the panel below that contains the description text. We did discuss not wrapping the whole thing in a panel though.

Reply:
@daveyholler feel free to push any changes direct to this branch!

Reply:
> This is the panel below that contains the description text. We did discuss not wrapping the whole thing in a panel though.

I believe that this particular line of code is a part of the panel that wraps the "whole thing", not the description panel within.